Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -72,44 +91,597 @@ screen initialState =
         eval state action
   }
 
-view
-  :: forall w
-  . (Action -> Effect Unit)
-  -> ST.DriverProfileScreenState
-  -> PrestoDOM (Effect Unit) w
+view  :: forall w. (Action -> Effect Unit)  -> ST.DriverProfileScreenState  -> PrestoDOM (Effect Unit) w
 view push state =
   frameLayout
     [ width MATCH_PARENT
     , height MATCH_PARENT
-    ][ linearLayout
-      [ height MATCH_PARENT
+    ][  linearLayout
+        [ height MATCH_PARENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , onBackPressed push (const BackPressed state)
+        , background Color.white900
+        , visibility if state.props.updateLanguages then GONE else VISIBLE
+        , padding $ PaddingBottom 24
+        ][  settingsView state push
+          , profileView push state]
+      , linearLayout
+        [ width MATCH_PARENT
+          , height MATCH_PARENT
+          , background Color.lightBlack900
+          , visibility if state.props.logoutModalView == true then VISIBLE else GONE
+          ][ PopUpModal.view (push <<<PopUpModalAction) (logoutPopUp state) ]
+        , if state.props.showLiveDashboard then showLiveStatsDashboard push state else dummyTextView 
+        -- , if state.props.updateDetails then updateDetailsView state push else dummyTextView
+        , if state.props.showGenderView || state.props.alternateNumberView then driverNumberGenderView state push else dummyTextView
+        , if state.props.removeAlternateNumber then PopUpModal.view (push <<<  RemoveAlternateNumberAC) (removeAlternateNumberConfig state ) else dummyTextView
+        , if state.props.enterOtpModal then enterOtpModal push state else dummyTextView
+        , if state.props.updateLanguages then updateLanguageView state push else dummyTextView]
+
+-- updateDetailsView :: forall w. ST.DriverProfileScreenState -> (Action -> Effect Unit )-> PrestoDOM (Effect Unit) w
+-- updateDetailsView state push = 
```

Comment:
Remove code, which is commented and not required

Reply:
This is required